### PR TITLE
refactor: capitalization and spelling

### PR DESCRIPTION
**Pull request by dejmedus**
Issue #38

Small changes to capitalization, spelling, and punctuation.

### CODE REVIEW

File: pages/about/components/OurStory.tsx [hunk 1]
```diff
@@ -68,19 +68,19 @@ export const OurStory: FC = () => {
...
+				content="You will be glad to hear that you are not alone there are many people out there too who do feel the same. WalkingPal is the antidote to all those problems of ours, it joins people like us who have a similar problem so that we can complete the journey easily together."
 				icon={footstep}
 			/>
 
 			<Story
 				rowReverse
 				imgSrc={story3}
-				content="By making walking more fun and enjoyable it's a great way to complete the last mile which is both cheap (free actually) and eco-friendly. Oh, and it's good for your health too. It's a great way to connect and meet new people, make new friends, relationships To explore and talk about the local food, places & culture that too completing your journey at the same time while walking together."
+				content="By making walking more fun and enjoyable it's a great way to complete the last mile which is both cheap (free actually) and eco-friendly. Oh, and it's good for your health too. It's a great way to connect and meet new people, make new friends, and build relationships. Explore and talk about local food, places, and culture while completing your journey walking together."
```
Comment (ashuvssut) [inline pages/about/components/OurStory.tsx:83]: Nice edits!

**Review by ashuvssut — APPROVED** [review]

**Comment by ashuvssut** [comment]
Wow nicely done! Fix the merge conflict rest looks perfect. Thanks for the PR

**Comment by dejmedus** [comment]
Hi, sorry this took so long. The "deleted/renamed" conflict threw me for a loop, but all should be okay to merge now

**Comment by ashuvssut** [comment]
Hi! actually you merge conflict resolution was incorrect. so I did it for you

If you are unsure how to fix a merge conflict, make sure you reach out to us in discord! Thanks for the PR

**Pull request merged**